### PR TITLE
create tcp client

### PR DESCRIPTION
**Pull request by vitalis-virtus**

### CODE REVIEW

File: tcpC.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	//fmt.Println("arguments from client", arguments)
+
+	//checking that a value for host:port was sent
+	if len(arguments) == 1 {
+		fmt.Println("Please provide host:port.")
+		return
+	}
+
+	// we save port in CONNECT variable
+	CONNECT := arguments[1]
```
Comment (penkova) [inline tcpC.go]: don't use uppercase for this variable

File: tcpC.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+		// we sent text message to the TCP server over the network using Fprintf()
+		fmt.Fprintf(c, text+"\n")
+
+		// we get server response message
+		message, _ := bufio.NewReader(c).ReadString('\n')
+		fmt.Print("SERVER-->: " + message)
+
+		// we terminate when user send STOP command to the tcp server
+		if strings.TrimSpace(string(text)) == "STOP" {
```
Comment (penkova) [inline tcpC.go]: Why are you using `string(test)`?`test` is already a string :)

File: tcpC.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+	//checking that a value for host:port was sent
+	if len(arguments) == 1 {
+		fmt.Println("Please provide host:port.")
+		return
+	}
+
+	// we save port in CONNECT variable
+	CONNECT := arguments[1]
+	//fmt.Println("CONNECT:", CONNECT)
```
Comment (penkova) [inline tcpC.go]: do you need this commented text?

File: tcpC.go (hunk 1 above)
Comment (vitalis-virtus) [reply tcpC.go]: fixed

File: tcpC.go (hunk 2 above)
Comment (vitalis-virtus) [reply tcpC.go]: fixed

File: tcpC.go (hunk 3 above)
Comment (vitalis-virtus) [reply tcpC.go]: removed this comment

**Review by penkova — APPROVED** [review]

**Pull request merged**